kong/db/declarative/init.lua

```diff
@@ -375,7 +375,7 @@ function declarative.load_into_db(entities, meta)
 end
 
 
-local function export_from_db(emitter, skip_ws)
+local function export_from_db(emitter, skip_ws, skip_disabled_services)
```

Comment:
```suggestion
local function export_from_db(emitter, skip_ws, skip_disabled_entities)
```

I would prefer to call it `skip_disabled_entities`. After this PR we have at least `plugins` and `services` that have `enabled` field.